ptxed/src/ptxed.c

```diff
@@ -46,7 +46,7 @@
 #include <inttypes.h>
 #include <errno.h>
 
-#include <xed-interface.h>
+#include <xed/xed-interface.h>
```

Comment:
Hmmm, not sure this helps.  If XED were packaged, I'd expect `#include <xed-interface.h>` to find it.  As long as you use XED from your local build, you anyway need to provide the path to the include files.  Whether you add '/xed' to the end of that path or not shouldn't make a difference.

Reply:
This came up as I was packaging libipt for Guix and it definitely helps.  CMIIW but I don't think there's any logic in libipt's CMake recipes to add `$PREFIX/include/xed` to the include paths, so headers are only looked up under `$PREFIX/include` (usually /usr/include) in most distros.

The reason this hasn't come up earlier is probably that other distributions packaging libipt don't enable the optional targets.  (Now what I'm unsure of is whether to include ptxed in the package.)

Reply:
The cmake logic uses user-supplied XED_INCLUDE and XED_LIB for local installations of XED.

Indeed, other distros are only packaging libipt.  The other components, ptdump and ptxed, are considered samples.  And ptseg is mostly a debug aid.  I haven't seen any distro packaging XED, either.  Are you also packaging XED or are you using a static build?

Reply:
Intel XED has been included in Guix System (and NixOS) since 2020.  User-supplied `XED_{INCLUDE,LIB}` should point to `XED_PREFIX/{include,lib}` IMHO.  Just because the header namespace happens to be the same name `xed` it doesn't mean that it doesn't exist.  One wouldn't ignore `llvm` or `SLD2` prefices, as the point of namespacing is for things to just work instead of having `C_INCLUDE_PATH=/usr/include/foo:/usr/include/bar:$C_INCLUDE_PATH`.

Reply:
So you don't need this patch anymore?

If you get requests to package ptdump and ptxed, let me know.

Reply:
XED indicated that this is how they plan to have their headers structure going forward.  I do hope that Linux distros agree on how to package XED.

Reply:
Would you mind signing your patch (git commit -s)?

Also, the convention is to tag the affected components in the commit message summary, which would be 'ptxed: ' in your case.

Reply:
No problem, I've updated the commit message to include the component and signed it off.